runtime/testy/studentAPI.py

```diff
@@ -9,12 +9,18 @@ def getValue(self, key):
     """Returns the value associated with key
     """
     # TODO: Actually use key
-    self.toManager.put([SM_COMMANDS.GET_VAL, [key]])
-    return self.fromManager.recv()
+    self.toManager.put([SM_COMMANDS.GET_VAL, key])
+    message = self.fromManager.recv()
+    if type(message) is StudentAPIValueError:
```

Comment:
is type() neccessary? I think there is a python function that checks is Object is an instance of Class
i think it might be simply
`if Object is Class:`